web/src/locales/da.json

```diff
@@ -263,29 +1570,204 @@
     "RU": {
       "zoneName": "Rusland"
     },
+    "RU-1": {
+      "countryName": "Rusland",
+      "zoneName": "European Russia and Ural"
+    },
+    "RU-2": {
+      "countryName": "Rusland",
+      "zoneName": "Siberia"
+    },
+    "RU-AS": {
+      "countryName": "Rusland",
+      "zoneName": "Øst Rusland"
+    },
+    "RU-EU": {
+      "countryName": "Rusland",
+      "zoneName": "Arctic"
+    },
+    "RU-FE": {
+      "countryName": "Rusland",
+      "zoneName": "Far East"
+    },
+    "RU-KGD": {
+      "countryName": "Rusland",
+      "zoneName": "Kaliningrad"
+    },
+    "RW": {
+      "zoneName": "Rwanda"
+    },
+    "SA": {
+      "zoneName": "Saudi-Arabien"
+    },
+    "SB": {
+      "zoneName": "Salomonøerne"
+    },
+    "SC": {
+      "zoneName": "Seychellerne"
+    },
+    "SD": {
+      "zoneName": "Sudan"
+    },
     "SE": {
       "zoneName": "Sverige"
     },
+    "SE-SE1": {
+      "countryName": "Sverige",
+      "zoneName": "Luleå / Nord"
+    },
+    "SE-SE2": {
+      "countryName": "Sverige",
+      "zoneName": "Sundsvall / Nord Central"
+    },
+    "SE-SE3": {
+      "countryName": "Sverige",
+      "zoneName": "Stockholm / Syd Central"
+    },
+    "SE-SE4": {
+      "countryName": "Sverige",
+      "zoneName": "Malmö / Syd"
```

Comment:
Do we want to include both names?

(this is not wrong but don't match the other languages)

Reply:
True, probably not